cve.py

```diff
@@ -30,3 +37,14 @@ def getCveDescription(self) -> str:
         """
         return self.description
 
+    def getCveSeverity(self) -> str:
+        """
+        :return:
+        """
+        return self.severity
+
+    def printCve(self):
+        print("CVE identifier: " + self.identifier)
```

Comment:
Should not have printing in a logical class. if you want to print data, the caller should print it.